cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/oidc/TokenAction.java

```diff
@@ -111,12 +111,16 @@ public void doAction() throws Exception {
         // (our config makes clients post the secret: "token_endpoint_auth_methods_supported: client_secret_post")
         String clientSecret = syncInput.getParameter("client_secret");
         if (clientID == null || clientSecret == null) {
-            sendError("invalid_client");
+            sendError("invalid_client: missing id or secret");
```

Comment:
I'm pretty sure the error messages are standard and can't be modified to add extra info.
https://openid.net/specs/openid-connect-core-1_0.html#TokenErrorResponse